src/Control/Session.php

```diff
@@ -119,6 +121,7 @@ class Session
     /**
      * @config
      * @var string
+     * @deprecated 6.1.0 Use `session.save_path` in ini configuration instead.
      */
     private static $session_store_path;
```

Comment:
For now this is used to override the ini config - but really this should be locked behind hosting configuration. If a developer doesn't have access to this configuration in their hosting environment, they probably shouldn't be changing where or how the file is stored in the filesystem.